gateway/1.9/modules/ROOT/pages/flex-conn-replica-caching.adoc

```diff
@@ -0,0 +1,26 @@
+= Configuring Replica Configuration Caching in Connected Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Enable replica configuration caching to have a Flex Gateway replica cache its gateway configuration in Redis shared storage. When configured, new replicas use the cached gateway configuration instead of downloading the configuration from Anypoint Platform. Caching a replica's configuration reduces scale-up time and increases resiliency.
+
+Replica caching works by:
+
+. When the first Flex Replica is created, it sees that their is no configuration in the Redis shared storage.
+. The first replica creates the group ID and downloads its configuration from Anypoint Platform. The replica then stores the configuration in Redis.
```

Comment:
I don't think the customer needs to know anything about a 'group ID'. This is an internal implementation detail.